app/controllers/assignments_controller.rb

```diff
@@ -2,7 +2,6 @@ class AssignmentsController < ApplicationController
   include RepositoryHelper
   include RoutingHelper
   include CriteriaHelper
-  include AnnotationCategoriesHelper
```

Comment:
Okay great. It seems like you can delete the `include` inside the test file as well.

Reply:
(done)